src/Persistence/Migrations/CrpgDbContextModelSnapshot.cs

```diff
@@ -30,32 +30,32 @@ protected override void BuildModel(ModelBuilder modelBuilder)
-            NpgsqlModelBuilderExtensions.HasPostgresEnum(modelBuilder, "weapon_class", new[] { "undefined", "dagger", "one_handed_sword", "two_handed_sword", "one_handed_axe", "two_handed_axe", "mace", "pick", "two_handed_mace", "one_handed_polearm", "two_handed_polearm", "low_grip_polearm", "arrow", "bolt", "cartridge", "bow", "crossbow", "stone", "boulder", "throwing_axe", "throwing_knife", "javelin", "pistol", "musket", "small_shield", "large_shield", "banner" });
+            NpgsqlModelBuilderExtensions.HasPostgresEnum(modelBuilder, "character_class", new[] { "archer", "cavalry", "crossbowman", "infantry", "mounted_archer", "peasant", "shock_infantry", "skirmisher" });
+            NpgsqlModelBuilderExtensions.HasPostgresEnum(modelBuilder, "clan_invitation_status", new[] { "accepted", "declined", "pending" });
+            NpgsqlModelBuilderExtensions.HasPostgresEnum(modelBuilder, "clan_invitation_type", new[] { "offer", "request" });
+            NpgsqlModelBuilderExtensions.HasPostgresEnum(modelBuilder, "clan_member_role", new[] { "leader", "member", "officer" });
+            NpgsqlModelBuilderExtensions.HasPostgresEnum(modelBuilder, "culture", new[] { "aserai", "battania", "empire", "khuzait", "looters", "neutral", "sturgia", "vlandia" });
+            NpgsqlModelBuilderExtensions.HasPostgresEnum(modelBuilder, "damage_type", new[] { "blunt", "cut", "pierce", "undefined" });
+            NpgsqlModelBuilderExtensions.HasPostgresEnum(modelBuilder, "game_mode", new[] { "crpg_battle", "crpg_captain", "crpg_conquest", "crpg_duel", "crpg_siege", "crpg_skirmish", "crpg_team_deathmatch", "crpg_unknown_game_mode", "crpgdtv" });
+            NpgsqlModelBuilderExtensions.HasPostgresEnum(modelBuilder, "item_slot", new[] { "body", "hand", "head", "leg", "mount", "mount_harness", "shoulder", "weapon0", "weapon1", "weapon2", "weapon3", "weapon_extra" });
+            NpgsqlModelBuilderExtensions.HasPostgresEnum(modelBuilder, "item_type", new[] { "arrows", "banner", "body_armor", "bolts", "bow", "bullets", "crossbow", "hand_armor", "head_armor", "leg_armor", "mount", "mount_harness", "musket", "one_handed_weapon", "pistol", "polearm", "shield", "shoulder_armor", "thrown", "two_handed_weapon", "undefined" });
+            NpgsqlModelBuilderExtensions.HasPostgresEnum(modelBuilder, "languages", new[] { "be", "bg", "cs", "da", "de", "el", "en", "es", "fi", "fr", "hr", "hu", "it", "lv", "nl", "no", "pl", "pt", "ro", "ru", "sr", "sv", "tr", "uk", "zh" });
+            NpgsqlModelBuilderExtensions.HasPostgresEnum(modelBuilder, "party_status", new[] { "following_party", "idle", "idle_in_settlement", "in_battle", "moving_to_attack_party", "moving_to_attack_settlement", "moving_to_point", "moving_to_settlement", "recruiting_in_settlement" });
+            NpgsqlModelBuilderExtensions.HasPostgresEnum(modelBuilder, "platform", new[] { "epic_games", "microsoft", "steam" });
```

Comment:
The enum reordering is likely to make the migration fail like the previous time. Why is this needed?

Reply:
The old way of defining an enumeration no longer works in ef as of version 9.0: https://www.npgsql.org/efcore/release-notes/9.0.html#breaking-changes

I made an empty migration on purpose so that the sorting order would not change, as advised by the EF developer: https://github.com/npgsql/efcore.pg/issues/3390

I tested this with my real db. But I suggest to test it on a stg

Reply:
@namidaka 
Ah, sorry, I see what you mean, I forgot to remove that from this file, thanks for noticing it

Reply:
I'll double-check everything again.

I'll deploy the current patch to myself, deploy the db, and then roll my updates and see if the enum order has changed, I'll see today

Reply:
@namidaka 

I've corrected the sorting and checked, everything seems fine.
We can look at the STG.

Reply:
@namidaka

After [communicating ](https://github.com/npgsql/efcore.pg/issues/3390) with EF core maintainer, I became convinced that you should never touch `CrpgDbContextModelSnapshot.cs` manually

The only thing that really affects the base is the migration, which I made empty. 

But the enum order in `CrpgDbContextModelSnapshot.cs`
must be updated because of the new EF (alphabetical order) we can't do anything about it.

I [tested](https://docs.google.com/spreadsheets/d/1jzVPrezX9aOl4pSJj18YUrfD0ZHMXmFqLf8SIMfwIrw/edit?gid=0#gid=0) the actual order of enums in `postgres` before and after this migration, the order has not changed